.github/workflows/automated-create-release-version.yml

```diff
@@ -9,15 +9,30 @@ on:
       partner_version:
         description: 'Partner version (e.g. 9.2.0)'
         required: true
+  repository_dispatch:
+    types: [create-release-branch]
 
 jobs:
   trigger:
     runs-on: ubuntu-latest
     steps:
+      # Since this workflow could be triggered by a user publicly or the internal cron job, we need to set the versions appropriately.
```

Comment:
aren't there ways to limit who triggers actions?

Reply:
oh this is intended behavior. it's just that depending on who triggers this, the input parsing is gonna look a little different. we allow this to be publicly trigger-able